games/t.yaml

```diff
@@ -487,18 +487,32 @@
   url: http://retrospec.sgn.net/game/tranzam
 
 - name: Tremulous
+  images:
+  - https://tremulous.net/screenshots/1.jpg
+  - https://tremulous.net/screenshots/2.jpg
+  - https://tremulous.net/screenshots/3.jpg
+  - https://tremulous.net/screenshots/6.jpg
   lang:
   - C
+  framework:
+  - SDL
```

Comment:
```suggestion
  - SDL2
```

Reply:
Latest stable Tremulous is 1.1 which came out in 2006. It was SDL at this time.